File: IDExamination/IDSeatingArrangement/IDSeatingArrangement/Invigilators.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IDSeatingArrangement
{
    class Invigilators
    {
        public string Id { get; set; }
        public string course { get; set; }
        public string course_name { get; set; }
        public string names { get; set; }
        public string t1date { get; set; }
        public string t2date { get; set; }
        public string time { get; set; }
        public Invigilators(string cid, string ccid, string t1,string t2,string ti,string name)
        {
            course = cid; course_name = ccid; t1date = t1;t2date = t2;time = ti; names = name;
        }
    }
}

File: Android/BPHC-ID/app/src/main/java/com/srujan/bphc_id/Invigilators.java
package com.srujan.bphc_id;

/**
 * Created by Srujan on 10/19/2015.
 */
public class Invigilators {
    public String Id;
    public String course;
    public String course_name;
    public String t1date;
    public String t2date;
    public String time;
    public String names;
    public Invigilators(String c1,String c2, String t1, String t2, String t, String name ) {
        course = c1;
        course_name=c2;
        t1date = t1;
        t2date = t2;
        time = t;
        names=name;
    }
}
